mysite/views.py

```diff
@@ -1,10 +1,17 @@
 """Views for activities app, handle html request."""
-from django.http import HttpRequest, JsonResponse
-from django.contrib.auth.decorators import login_required
-from django.conf import settings
+from django.http import HttpRequest
+from allauth.socialaccount.models import SocialAccount
+from .settings import OAUTH_PROVIDER
+from rest_framework import decorators, response
 
 
-def is_authen(request: HttpRequest) -> JsonResponse:
-    """Return Json which inform that user is login right now."""
-    print(request.user.is_authenticated)
-    return JsonResponse({"is_authen": request.user.is_authenticated})
+@decorators.api_view(["GET"])
+def profile_picture_view(request: HttpRequest) -> response.Response:  # pragma: no cover
+    """Return profile picture url from Google account."""
+    user = request.user
+    try:
+        social_account = SocialAccount.objects.get(user=user, provider=OAUTH_PROVIDER)
```

Comment:
You can remove provider, we got only 1. If you consider remove this you may need to remove OAUTH_PROVIDER from `settings.py` and `sample.env` also

Reply:
@opxz7148 [fixed it.](https://github.com/OmegaOoh/ku-tangtee/pull/64/commits/521eafdb83c009e0b5591ab5e9a4bec6137b6cd4)